nifi-nar-bundles/nifi-standard-bundle/pom.xml

```diff
@@ -265,7 +290,7 @@
             <dependency>
                 <groupId>com.networknt</groupId>
                 <artifactId>json-schema-validator</artifactId>
-                <version>1.1.0</version>
+                <version>1.0.87</version>
```

Comment:
This change should be reverted.

Reply:
Kind of weird as I updated main and then on my branch I rebased with main, I am not sure why all these changes were not picked up.